astro/src/content/quickstarts/quickstart-javascript-express-web.mdx

```diff
@@ -0,0 +1,320 @@
+
+### Setup Application
+
+In this section, you'll set up a basic Golang application with a single page.
+
+#### Setup Your Environment
+
+Create a new directory to hold your application, and go into it.
+
+```shell
+mkdir changebank
+cd changebank
+```
+
+To keep track of changes, initialize your directory with git.
+
+```shell
+git init
+```
+
+Create a file called `.gitignore` so that you can exclude files from your git repo.
+
+`.gitignore`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/.gitignore"}
+  lang="text" />
+
+
+Create a file called `package.json` with the below dependencies and scripts.
+
+`package.json`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/package.json"}
+  lang="json" />
+
+Install dependencies using npm.
+
+```shell
+npm install
+```
+
+Create a new directory to hold your source code, and go into it.
+
+```shell
+mkdir src
+cd src
+```
+
+#### Setup New Application via API
+
+<Aside type="tip">
+Although you could also do all of this in the kickstart, this example is meant to show off how to use the SDK on the server for administrative purposes. You could also make any of these changes within the admin UI. You can do this even after the instance is running unlike with kickstart, that needs to happen on a fresh install.
+</Aside>
+
+In this setup you will:
+
+1. Update the Tenant's issuer to `http://localhost:9011`
+1. Generate an RSA Key for the App
+1. Create a new Application named `JSExampleApp` using the new RSA key.
+1. Find the User with email `richard@example.com`
+1. Update `richard@example.com`'s fullName property to `Richard Bray`
+1. Register the User `Richard Bray` with the new application
+
+Make sure you are still in the directory `complete-application/src`
+
+Create a file called `setup.ts` with the below code.
+
+`setup.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/setup.ts"}
+  lang="ts" />
+
+Execute this setup by calling an npm script.
+
+```shell
+npm run setup
+```
+
+## Create the Express Application
+
+In order for our application to look good we have some files that are already made, copy `complete-application/static` and `complete-application/templates` to the `changebank` directory. These static files are used to create the "beautiful" Changebank application UI.
+
+<Aside type="note">
+You can choose to examine the full example of the running code now, or skip below and walk through each step of the process.
+</Aside>
+
+### Full Example
+
+Make sure once again that you are in the directory `complete-application/src`.
+Create a file called `index.ts` with the below code.
+
+`index.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts" />
+
+#### Run the App!
+You should now be able to start your Express application with 
+
+```shell
+npm run dev
+```
+
+### Walkthrough Example
+
+Make sure once again that you are in the directory `complete-application/src`.
```

Comment:
Shouldn't we go back to `changebank/src` again?

Reply:
yes